lego_image.py

```diff
@@ -1,26 +1,46 @@
+import argparse
 import itertools
 from math import sqrt
 
 from PIL import Image
 
-from lego_tuple import Lego_return_tuple
+from lego_tuple import Lego_colours
 
-im = Image.open("joker.jpg") # what is a good way to pass argument from main project.py file?
-# rgb_im = im.convert("RGB")
-size = (48, 48)
+COLORS = [i.RGB for i in Lego_colours]
+parser = argparse.ArgumentParser(description="Change an image to a LEGO ART set")
```

Comment:
you could break this out into a main.py = entry point, when you put the code into a package, you can then rename it to __main__.py to make it an entry point to the package (that is, you can call it with python -m package), see:

<img width="1476" alt="image" src="https://user-images.githubusercontent.com/387927/166719970-f520c359-bbba-452d-9863-1c2fef75f00e.png">